cmd/k3s-release/create-tags.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/rancher/ecm-distro-tools/release/k3s"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+func CreateTagsCommand() cli.Command {
+	return cli.Command{
+		Name:   "create-tags",
+		Usage:  "Create tags from kubernetes repo",
+		Flags:  rootFlags,
+		Action: CreateTags,
+	}
+}
+
+func CreateTags(c *cli.Context) error {
+	ctx := context.Background()
```

Comment:
We should create a ctx value in func main and pass it anywhere a ctx value is needed.